### DISCUSSION TITLE
Limit relationship data to display in table component

### DISCUSSION THREAD

**Discussion by Winniepoom** [Budibase/budibase#11526] [Help] [answered]
Hello I had a question I had an relationship issue id and it displaying in table, What if I want it to be display just 3 recent issue id how I'm suppose to do that?
![image](https://github.com/Budibase/budibase/assets/107398226/42110845-429f-4794-bef5-e6a46cc129cc)

**Comment by andz-bb** [comment, marked as answer]
hey @Winniepoom

To achieve this I added a static formula column to my table and used this JavaScript to return an array of the IDs of the last three related rows:
```js
const jobs = $("Jobs") || [] // Jobs is the name of my relationship column
return jobs.slice(-3).map(job => job["Job ID"]) // Job ID is the display column of the related rows
```

This returns an array of IDs like `[4,5,6]` which I could format more nicely as a string using `.join(", ")`. I did this in the column settings of my table component but you could also add it to the formula column itself.

Here's the final result in my table - hope this helps!

<img width="532" alt="image" src="https://github.com/Budibase/budibase/assets/110921612/d925d3bc-e162-4cb0-85d0-d3ca24c8d2a9">

**Comment by ShinJustinHolly3317** [comment]
How can I show all items instead of showing "+1 more"
